In this module we were tasked with exceptions in Python programming language.
